### ISSUE TITLE
SyntaxError: from __future__ imports must occur at the beginning of the file

### ISSUE THREAD

**Issue by VladimirChukharev** [enthought/pyface#188]
Trying to port pyface to FreeBSD, during installation I get the following error.
```
...
byte-compiling /usr/ports/devel/py-pyface/work/stage/usr/local/lib/python2.7/site-packages/pyface/resource_manager.py to resource_manager.pyc
byte-compiling /usr/ports/devel/py-pyface/work/stage/usr/local/lib/python2.7/site-packages/pyface/list_box_model.py to list_box_model.pyc
byte-compiling /usr/ports/devel/py-pyface/work/stage/usr/local/lib/python2.7/site-packages/pyface/ipython_widget.py to ipython_widget.pyc
  File "/usr/local/lib/python2.7/site-packages/pyface/ipython_widget.py", line 27
    You need to have ipython >= 0.9 installed to use the ipython widget.'''
SyntaxError: from __future__ imports must occur at the beginning of the file

byte-compiling /usr/ports/devel/py-pyface/work/stage/usr/local/lib/python2.7/site-packages/pyface/system_metrics.py to system_metrics.pyc
...
```
To fix the problem, I use the following patch.
```
 $ cat files/patch-pyface_ipython__widget.py 
--- pyface/ipython_widget.py.orig       2016-01-06 22:56:14 UTC
+++ pyface/ipython_widget.py
@@ -14,6 +14,9 @@
 """ The implementation of an IPython shell. """
 
 
+from __future__ import absolute_import
+
+
 # Import the toolkit specific version.
 try:
     import IPython.frontend
@@ -24,7 +27,5 @@ Could not load the Wx frontend for ipyth
 You need to have ipython >= 0.9 installed to use the ipython widget.'''
 
 
-from __future__ import absolute_import
-
 from .toolkit import toolkit_object
 IPythonWidget= toolkit_object('ipython_widget:IPythonWidget')
```
Hope this is useful for somebody.
Vladimir

PS. pyface-5.0.0.tar.gz
```
 $ python --version
Python 2.7.11
 $ uname -a
FreeBSD vovaasus 9.3-STABLE FreeBSD 9.3-STABLE #63 r293184M: Wed Jan  6 15:16:38 EET 2016     root@vovaasus:/usr/obj/usr/src/sys/VOVA  amd64
```

**Comment by jakirkham** [comment]
Seeing the exact same problem.

**Comment by jakirkham** [comment]
cc @grlee77

**Comment by grlee77** [comment]
This seems to have already been fixed in current master by 06d85f8758321b4aaca4a52e3df4ee357c889516

**Comment by jakirkham** [comment]
Thanks for narrowing that down. Appears to be helping our case. Maybe this issue can be closed.

**Comment by VladimirChukharev** [comment]
OK, fixed in [06d85f8](https://github.com/enthought/pyface/commit/06d85f8758321b4aaca4a52e3df4ee357c889516)
Thanks!
